imports/client/components/OwnProfilePage.tsx

```diff
@@ -280,6 +282,10 @@ enum OwnProfilePageSubmitState {
   ERROR = 'error',
 }
 
+const PageContainer = styled.div`
+  max-width: 800px;
+`;
+
```

Comment:
Instead of defining our own class here and in `SectionContent`, thoughts on using Bootstrap's/`react-bootstrap`'s `Container`, which serves a similar function?

Reply:
Hmmm.  I think the centering behavior might make sense in some contexts, and I could probably use it here.

For SetupPage specifically I think I want the max-width to be narrower than react-bootstrap's largest breakpoint of 960px.